Changelog.md

```diff
@@ -0,0 +1,60 @@
+# Toolkit Changelog
+
+## Process for updating this change log (Proposal)
+This change log should be updated as part of a PR if the work is worth noting (most of them should be). If unsure, always add an entery here for any PR targeted for the next release. It's easier to remove than add an entry at final review time for the next release.
```

Comment:
```suggestion
This changelog should be updated as part of a PR if the work is worth noting (most of them should be). If unsure, always add an entry here for any PR targeted for the next release. It's easier to remove than add an entry at final review time for the next release.
```

Reply:
Spelling suggestions, these are not blocking.

Reply:
Good catch!